### PR TITLE
[team-soju#3926] log level and logs at end of test property

### PR DESCRIPTION
**Pull request by maartenjanvangool**

### CODE REVIEW

File: src/main/java/io/orangebeard/client/OrangebeardV2Client.java [hunk 1]
```diff
@@ -116,6 +117,20 @@ public void log(Log log) {
         }
     }
 
+    @Override
+    public void log(Set<Log> logs) {
```
Comment (tcnh) [inline src/main/java/io/orangebeard/client/OrangebeardV2Client.java:121]: Should we let the `log(Log log)` function just call this one using `log(Collections.singleton(log)`? Any reason to keep both explicit calls to the /log endpoint?

File: src/main/java/io/orangebeard/client/OrangebeardProperties.java [hunk 2]
```diff
@@ -140,4 +194,27 @@ private UUID lookupUUIDWithDefault(OrangebeardProperty property, Function<String
...
+                return 0;
+            case info:
+                return 1;
+            case warn:
+                return 2;
+            case error:
+            case fatal:
+                return 3;
+        }
+        return 1;
```
[the remark this answers is not in the document]
Comment (maartenjanvangool) [reply src/main/java/io/orangebeard/client/OrangebeardProperties.java:218]: Yeah... I thought making all the branches explicit was more clear (to me).

File: src/main/java/io/orangebeard/client/OrangebeardV2Client.java (hunk 1 above)
Comment (maartenjanvangool) [reply src/main/java/io/orangebeard/client/OrangebeardV2Client.java:121]: Good point.

File: src/main/java/io/orangebeard/client/OrangebeardProperties.java [hunk 3]
```diff
@@ -140,4 +194,27 @@ private UUID lookupUUIDWithDefault(OrangebeardProperty property, Function<String
         }
         return attributes;
     }
+
+    public boolean logShouldBeDispatchedToOrangebeard(LogLevel individualLogLevel) {
```
[the remark this answers is not in the document]
Comment (maartenjanvangool) [reply src/main/java/io/orangebeard/client/OrangebeardProperties.java:198]: Is for use in a listener itself. I am using this in the toolchain listener.

**Pull request merged**